osidb/tests/conftest.py

```diff
@@ -66,6 +66,11 @@ def embargo_read_group():
     return generate_acls([settings.EMBARGO_READ_GROUP])
 
 
+@pytest.fixture
```

Comment:
I know this is not your fault but this is a weird set of fixtures. I do not understand the purpose of having both the singular and plural once. The singular once does not even seem to be used for anything than creating the plural once. Also one can easily make a type omitting **s** and will get an existing but different thing. When we are refactoring this I would be for merging the singular and plural ones together.

Reply:
I agree with you. In my case, I just want to avoid writing `[uuid.UUID(acl) for acl in generate_acls([settings.INTERNAL_WRITE_GROUP])]` every time I need it.

I was thinking about refactoring but was unsure whether I understood all ACLs usage correctly, so I left it for later.